src/PlaywrightSharp/Playwright.cs

```diff
@@ -55,23 +55,28 @@ internal Playwright(IChannelOwner parent, string guid, PlaywrightInitializer ini
         /// <summary>
         /// Returns a list of devices to be used with <see cref="IBrowser.NewContextAsync(BrowserContextOptions)"/>.
         /// </summary>
-        public IReadOnlyDictionary<string, DeviceDescriptor> Devices => _devices;
+        public IReadOnlyDictionary<string, DeviceDescriptor> Devices { get => _devices; set => throw new NotSupportedException(); }
```

Comment:
hmmmm

Reply:
I wonder if *all* of our generated properties should just be read-only?

Reply:
I think so. The SetTimeout is still a method right? That would be the only one.

Reply:
I will have to revisit this _later_. I need the properties to be generated when we have a serializable object (i.e. all the models). I don't want to block this PR in the mean time, though. As we've talked about a few times, when we finish the fist pass, we'll probably just re-add all the generated interfaces to fix the small discrepancies that are introduced as we change stuff on the generator. Let's keep this as is, then, for now, and we'll fix it before release.

Reply:
https://github.com/microsoft/playwright-sharp/issues/1286